release_tester/arangodb/installers/tar.py

```diff
@@ -0,0 +1,163 @@
+        version = ".".join(version.split(".")[:3])
+        self.semver = semver.VersionInfo.parse(version)
+
+        cfg.have_system_service = False
+        cfg.baseTestDir = Path('/tmp')
+        cfg.installPrefix = None
+        cfg.bin_dir = None
+        cfg.sbin_dir = None
+        cfg.real_bin_dir = None
+        cfg.real_sbin_dir = None
+        cfg.localhost = 'ip6-localhost'
+
+        cfg.logDir = None
+        cfg.dbdir = None
+        cfg.appdir = None
+        cfg.cfgdir = None
+
+
+        super().__init__(cfg)
+
+    def calculate_package_names(self):  
+        # Are those required to be stored in the cfg?
+        enterprise = 'e' if self.cfg.enterprise else ''
+        # package_version = '1'
+        architecture = 'linux'
+
+        self.desc = {
+            "ep"   : enterprise,
+            "ver"  : self.cfg.version,
+            # "ver"  : package_version,
+            "arch" : architecture
+        }
+        self.server_package = 'arangodb3{ep}-{arch}-{ver}.tar.gz'.format(**self.desc)
+        # self.server_package = 'arangodb3{ep}_{cfg}-{ver}_{arch}.deb'.format(**desc)
+        # self.client_package = 'arangodb3{ep}-client_{cfg}-{ver}_{arch}.deb'.format(**desc)
+        # self.debug_package = 'arangodb3{ep}-dbg_{cfg}-{ver}_{arch}.deb'.format(**desc)
+
+        self.cfg.installPrefix = Path("/tmp")/'arangodb3{ep}-{ver}'.format(**self.desc)
+        self.cfg.bin_dir = self.cfg.installPrefix / "usr" / "bin"     # /usr/bin
+        self.cfg.sbin_dir = self.cfg.installPrefix / "usr" / "sbin"   # /usr/sbin
+        self.cfg.real_bin_dir = self.cfg.bin_dir
+        self.cfg.real_sbin_dir = self.cfg.sbin_dir
+
+    def check_service_up(self):
+        pass
+        # time.sleep(1)    # TODO
+        # print("I am here in: deb.py line: 69")
+        # return True
+
+    def start_service(self):
+        pass
+        # print("I am here in: deb.py line: 73")
+
+        # startserver = pexpect.spawnu('service arangodb3 start')
+        # logging.debug("waiting for eof")
+        # startserver.expect(pexpect.EOF, timeout=30)
+        # while startserver.isalive():
+        #     print('.', end='')
+        #     if startserver.exitstatus != 0:
+        #         raise Exception("server service start didn't"
+        #                         "finish successfully!")
+        # time.sleep(0.1)
+        # self.instance.detect_pid(1) # should be owned by init
+
+    def stop_service(self):
+        pass
+
+        # print("I am here in: deb.py line: 88")
+
+        # stopserver = pexpect.spawnu('service arangodb3 stop')
+        # logging.debug("waiting for eof")
+        # stopserver.expect(pexpect.EOF, timeout=30)
+        # while stopserver.isalive():
+        #     print('.', end='')
+        #     if stopserver.exitstatus != 0:
+        #         raise Exception("server service stop didn't"
+        #                         "finish successfully!")
+
+    def upgrade_package(self):
+       
```

Comment:
replace this blank line by """ + the comment you had below - python function documentation.